app/assets/stylesheets/_variables.scss

```diff
@@ -16,20 +16,54 @@ $small-spacing: $base-spacing / 2;
 $base-z-index: 0;
 
 // Colors
-$blue: #477dca;
-$dark-gray: #333;
-$medium-gray: #999;
-$light-gray: #ddd;
-
-// Font Colors
-$base-background-color: #fff;
-$base-font-color: $dark-gray;
-$action-color: $blue;
-
-// Border
-$base-border-color: $light-gray;
+$dark-color: #333;
+$light-color: #fff;
+
+// Background
+$base-background: #fff;
+$base-background-inset: darken($base-background, 4%);
+$base-background-inset-shadow: inset 0 0 2px darken($base-background, 20%);
+$base-font-color-on-background: $dark-color;
+
+// Outline on background
+$base-outline-color: darken($base-background, 15%);
+$base-outline: 1px solid $base-outline-color;
+
+// Box
+$base-box-background: darken($base-background, 3%);
+$base-border-color: darken($base-box-background, 15%);
 $base-border: 1px solid $base-border-color;
+$base-box-background-inset: darken($base-box-background, 5%);
+$base-font-color-on-box: #000;
+$base-box-shadow: 0 1px 2px transparentize($base-font-color-on-box, 0.8);
+$base-box-shadow-inset: inset 0 2px 2px darken($base-box-background, 10%);
+
+// Notification Colors
+$alert-color: #fff6bf;
+$error-color: #fbe3e4;
+$notice-color: #e5edf8;
+$success-color: #e6efc2;
+
+// Action Colors
+$action-color-on-background: #477dca;
+$action-color-on-box: #477dca;
 
 // Forms
+$form-background: #fff;
+$form-border-color: $base-border-color;
+$form-border: 1px solid $form-border-color;
 $form-box-shadow: inset 0 1px 3px rgba(#000, 0.06);
-$form-box-shadow-focus: $form-box-shadow, 0 0 5px adjust-color($action-color, $lightness: -5%, $alpha: -0.3);
+$form-box-shadow-focus: $form-box-shadow, 0 0 5px adjust-color($action-color-on-background, $lightness: -5%, $alpha: -0.3);
+
+// Media
+$medium-screen: em(640);
+$large-screen: em(860);
```

Comment:
They are needed in Refills to avoid repetition, but I understand if Bitters doesn't want them.